DependencyInjection/BernardExtension.php

```diff
@@ -130,4 +134,17 @@ private function registerListeners(array $config, ContainerBuilder $container)
             }
         }
     }
+
+    public function configureProfiler(ContainerBuilder $container, array $config)
+    {
+        if (isset($config['profiler']['enabled']) && !$config['profiler']['enabled']) {
+            return;
+        }
+
+        $definition = new Definition(ProducerCollector::class);
```

Comment:
Oh you mean the other way around. I first had it like that but then struggled to make it conditional (wasn't much of an issue) but I refactored it to php then. I don't know if there is a preference to xml or php but I cna change it back!